Recipes/Recipes/AuthProfile/AuthPresenter.swift

```diff
@@ -3,13 +3,116 @@
 
 import UIKit
 
+/// Протокол для view экрана авторизации
+protocol AuthView {
```

Comment:
```suggestion
protocol AuthViewProtocol {
```
у всех протоколов окончание Protocol